src/docs/01_introduction_and_goals.adoc

```diff
@@ -62,10 +62,10 @@ A table with quality goals and concrete scenarios, ordered by priorities
 [options="header", cols="0,0,4,3"]
 |===
 |Priority| Goal         | Motivation | Scenarios
-|1| Usability |The app must be easy to use by non-technical people. | 
-|2| Privacy    | Users must be the owners of their own data. |
-|3| Effectiveness | Users must be able to share their routes according to the functionalities of the system. |
-|4| Interoperability | The system must be able to communicate with other similar SOLID apps |
+|1| Usability |The app must be easy to use by non-technical people. | A new user must be able to understand the application and use it without advanced knowledge in the software engineering field.
+|2| Privacy    | Users must be the owners of their own data. | Users must be able to locate their data wherever they want to and have full control of it.
+|3| Effectiveness | Users must be able to share their routes according to the functionalities of the system. | The routes of a user must be able to be shared through the application.
+|4| Interoperability | The system must be able to communicate with other similar SOLID apps | Routes created with other SOLID apps must be able to be loaded to the system and be able to be worked with.
```

Comment:
Here, motivation and scenarios might be something similar, do you think we should merge them somehow?

Reply:
The scenarios should be reformulated in order to show real-world situations